core/kazoo_globals/src/kazoo_globals_init.erl

```diff
@@ -0,0 +1,22 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2016, 2600Hz
+%%% @doc
+%%% Wait for Globals
+%%% @end
+%%% @contributors
+%%%   Luis Azedo
+%%%-------------------------------------------------------------------
+-module(kazoo_globals_init).
+
+-export([start_link/0]).
+
+-include("kazoo_globals.hrl").
+
+start_link() ->
+    wait_for_globals('false').
+
+wait_for_globals('true') ->
```

Comment:
could be, but this is a different process and we are not expecting other processing from same process. added 1 second sleep